netman/adapters/switches/cisco.py

```diff
@@ -23,6 +23,8 @@
     BadVrrpPriorityNumber, VrrpDoesNotExistForVlan, BadVrrpTimers, BadVrrpTracking, UnknownDhcpRelayServer, DhcpRelayServerAlreadyExists, \
     VlanAlreadyExist, UnknownBond
 from netman.core.objects.interface import Interface
+from netman.core.objects.interface_states import OFF
+from netman.core.objects.interface_states import ON
```

Comment:
did you import manually?  try auto-organizing imports they should fall on the same line